deploy/crds/nmstate.io_v1alpha1_nodenetworkconfigurationenactment_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: nmstate.io/v1alpha1
```

Comment:
Please make this into a proper example

Reply:
Since it's read only does it make sense to have examples ?

Reply:
Examples are interesting for newcomers, but hard to maintain and keep up-to-date. Up to you, either make it a proper example or drop it.

Reply:
Let's play nice with operator-sdk and keep them.